README.rst

```diff
@@ -144,11 +144,10 @@ See ``make help``.
 ``fixtures/rpm-unsigned``
     The ``createrepo`` and ``modifyrepo`` executables must be available.
 
-``fixtures/rpm-richnweak-deps/srpms``
-    The ``mock`` executable must be available.
-
 ``fixtures/rpm-richnweak-deps``
-    See ``fixtures/rpm-richnweak-deps/srpms``.
+    The ``mock``, ``createrepo`` executable must be available.
+    All users that are to use mock must be added to the mock group.
+    ``usermod -a -G mock [User name]``
```

Comment:
This would be even more useful if one could execute the given code in some environment, such as in a bash script. Maybe something like `usermod --append --groups mock "$(whoami)"` or `gpasswd --add "$(whoami)" mock`.